boot/cmdline_test.go

```diff
@@ -413,7 +413,7 @@ func (s *kernelCommandLineSuite) TestBootVarsForGadgetCommandLine(c *C) {
 		sf := snaptest.MakeTestSnapWithFiles(c, gadgetSnapYaml, append([][]string{
 			{"meta/snap.yaml", gadgetSnapYaml},
 		}, tc.files...))
-		vars, err := boot.BootVarsForTrustedCommandLineFromGadget(sf)
+		vars, err := boot.BootVarsForTrustedCommandLineFromGadget(sf, "")
```

Comment:
this needs to test also the cases when opt is not empty, no?

Reply:
Tests added